packages/block-editor/src/store/reducer.js

```diff
@@ -1650,6 +1650,15 @@ export function highlightedBlock( state, action ) {
 	return state;
 }
 
+export function hoveredBlocks( state = [], action ) {
+	switch ( action.type ) {
+		case 'HOVERED_BLOCKS_SET':
```

Comment:
`Non-blocking`:

This name gave me pause. It looks like most other action types use the convention of VERB_NOUN (Ex. SELECT_BLOCK, SET_NAVIGATION_MODE, etc.) Is there a reason we don't do the same here? 

The other reason I bring this up is because I thought we were describing a "set" in the sense of the [abstract data type](https://en.wikipedia.org/wiki/Set_(abstract_data_type)), and I spent a while wondering why state management wasn't implemented with [a javascript set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set).

Reply:
good call, updated!